pagopa/io-app
# chore: [IOPLT-787] Upgrade react-native-image-picker to fix android policy issue

> [!note]
> This PR depends on #6348 

## Short description
This PR upgrades `react-native-image-picker` to 7.2.2 in order to fix the new google play policy on the `READ_MEDIA_IMAGES` permission

## List of changes proposed in this pull request
- Upgrade image picker library to latest available

## How to test
Check the pick of an image on Barcode screen on both OS to check everything's fine
